sources/StdlibUpgradeScripts.move

```diff
@@ -119,5 +122,18 @@ module StdlibUpgradeScripts {
             i = i + 1;
         }
     }
+
+    /// Burned by user account
+    const ERR_NOT_RIGHT_ADDRESS: u64 = 101;
+    public entry fun burn_illegal_token(sender: signer, amount: u128) {
+        if (ChainId::is_main()) {
```

Comment:
Does it mean that the illegal token will transfer to this address?

Reply:
yes